pandas/core/generic.py

```diff
@@ -5716,24 +5716,75 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Trim the caller's values below a given `threshold`.
+
+        Elements below the `threshold` will be changed to match the
+        `threshold` value(s).
 
         Parameters
         ----------
-        threshold : float or array_like
-        axis : int or string axis name, optional
+        threshold : float or array-like
+            Lower value(s) to which the input value(s) will be trimmed.
+        axis : {0 or 'index', 1 or 'columns', None}, default None
             Align object with threshold along the given axis.
         inplace : boolean, default False
-            Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+            Whether to perform the operation in place on the data.
 
-        See Also
-        --------
-        clip
+            .. versionadded:: 0.21.0
 
```

Comment:
Sorry if I forgot some discussion about it, but don't we want a `See Also` section? I think `clip` and `clip_upper` should be listed.

Reply:
Yes good spot we definitely still want this - @adatasetaday can you add back in?

I know we had some back and forth about `Series.clip_lower` being self-referential with the shared doc system. If you find that confusing just remove that one entry, but all the rest should remain